__tests__/app.test.js

```diff
@@ -106,3 +105,48 @@ describe.only('GET/api/articles', () => {
         });
     });
 });
+describe('GET/api/articles/:article_id/comments', () => {
+        test('returns an array of comments for the given article id', () => {
+            return request(app)
+            .get('/api/articles/9/comments')
+            .expect(200)
+            .then((response) => { 
+                expect(response.body.comments.length).toBe(2);
+                response.body.comments.forEach((comment) => {
+                    expect(typeof comment.comment_id).toBe('number');
+                    expect(typeof comment.votes).toBe('number');
+                    expect(typeof comment.created_at).toBe('string');
+                    expect(typeof comment.author).toBe('string');
+                    expect(typeof comment.body).toBe('string');
+                    expect(comment.article_id).toBe(9);
+                })
+            })
+        });
+        test('returns an array of comments for the given article id sorted by created_at in decending order', () => {
+            return request(app)
+            .get('/api/articles/9/comments')
+            .expect(200)
+            .then((response) => { 
+                expect(response.body.comments).toBeSortedBy('created_at', {
+                    descending: true,
+                });
+            })
+        });
+        test('returns error 404 when article id is valid but does not exist in the database', () => {
+            return request(app)
+            .get('/api/articles/100/comments')
+            .expect(404)
+            .then((response) => { 
+                expect(response.body.msg).toBe('No comments found for 100');
```

Comment:
This error message isn't quite descriptive enough as it implies that the problem is that there are no comments, not that there is no article